app/graphql/schema.rb

```diff
@@ -13,4 +13,8 @@ class Schema < GraphQL::Schema
   use ComplianceTimeout, max_seconds: 20
   query Types::Query
   mutation Types::Mutation
+  lazy_resolve(Promise, :sync)
```

Comment:
It allows lazy loaders like CollectionLoader or RecordLoader to resolve the promises lazily (after all IDs are collected) so that the SQL queries are batched. https://github.com/rmosolgo/graphql-ruby/blob/master/guides/schema/lazy_execution.md#example-batched-find